docker-compose.override.yml

```diff
@@ -0,0 +1,6 @@
+version: '3.8'
```

Comment:
These kind of files are usually excluded from version control. Otherwise it just can be part of the main file.